### ISSUE TITLE
"Try It" section in readme unsuccessful

### ISSUE THREAD

**Issue by shaunlebron** [adzerk/boot-cljs#58]
In an empty directory, I ran the "Try It" section.  

```
$ mkdir src
$ echo -e '(ns foop)\n(.log js/console "hello world")' > src/foop.cljs
$ boot -s src -d adzerk/boot-cljs cljs

Writing main.cljs.edn...
                                  clojure.lang.ExceptionInfo: java.util.concurrent.ExecutionException: org.sonatype.aether.resolution.DependencyResolutionException: Could not find artifact org.clojure:clojurescript:jar:0.0.0 in clojars (http://clojars.org/repo/)
    data: {:file
           "../var/folders/n9/3v0_clqj0f3gm3956_06hq680000gp/T/boot.user36358634228092199.clj",
           :line 5}
                     java.util.concurrent.ExecutionException: java.util.concurrent.ExecutionException: org.sonatype.aether.resolution.DependencyResolutionException: Could not find artifact org.clojure:clojurescript:jar:0.0.0 in clojars (http://clojars.org/repo/)
                     java.util.concurrent.ExecutionException: org.sonatype.aether.resolution.DependencyResolutionException: Could not find artifact org.clojure:clojurescript:jar:0.0.0 in clojars (http://clojars.org/repo/)
org.sonatype.aether.resolution.DependencyResolutionException: Could not find artifact org.clojure:clojurescript:jar:0.0.0 in clojars (http://clojars.org/repo/)
    result: #<DependencyResult [adzerk:boot-cljs:jar:0.0-2814-1 < clojars (http://clojars.org/repo/, releases+snapshots), null < null]>
  org.sonatype.aether.resolution.ArtifactResolutionException: Could not find artifact org.clojure:clojurescript:jar:0.0.0 in clojars (http://clojars.org/repo/)
     result: #<ArtifactResult adzerk:boot-cljs:jar:0.0-2814-1 < clojars (http://clojars.org/repo/, releases+snapshots)>
    results: [#<ArtifactResult adzerk:boot-cljs:jar:0.0-2814-1 < clojars (http://clojars.org/repo/, releases+snapshots)> #<ArtifactResult null < null>]
      org.sonatype.aether.transfer.ArtifactNotFoundException: Could not find artifact org.clojure:clojurescript:jar:0.0.0 in clojars (http://clojars.org/repo/)
      artifact: #<DefaultArtifact org.clojure:clojurescript:jar:0.0.0>
    repository: #<RemoteRepository clojars (http://clojars.org/repo/, releases+snapshots)>
     org.sonatype.aether.connector.wagon.WagonRepositoryConnector$4.wrap  WagonRepositoryConnector.java: 947
     org.sonatype.aether.connector.wagon.WagonRepositoryConnector$4.wrap  WagonRepositoryConnector.java: 941
org.sonatype.aether.connector.wagon.WagonRepositoryConnector$GetTask.run  WagonRepositoryConnector.java: 669
       org.sonatype.aether.util.concurrency.RunnableErrorForwarder$1.run    RunnableErrorForwarder.java:  60
                                                                     ...
```

Looks like it's looking for clojurescript 0.0.0 for some reason.

Here's my boot version:

```
$ boot -V
#https://github.com/boot-clj/boot
#Mon Feb 23 22:28:14 CST 2015
BOOT_CLOJURE_VERSION=1.6.0
BOOT_VERSION=2.0.0-rc9
```

**Comment by martinklepsch** [comment]
Thanks for reporting, I added a PR to fix this: #59

**Comment by shaunlebron** [comment]
Cool, thanks.  Can we get the new version published?

**Comment by martinklepsch** [comment]
Should be released with `0.0-2814-3`.

**Comment by shaunlebron** [comment]
Sweet, it works now.  Thanks
